### ISSUE TITLE
Personal and Universal views 

### ISSUE THREAD

**Issue by Blandine-AA** [hhaccessibility/hhaccessibility.github.io#109]
This is how this views are supposed to work:

**Universal view**
- for anyone who logs in as a guest or who doesn't have his accessibility needs set in his profile
- Accessibility calculation is done based on ALL criteria

**Personal view**
- For the user who has signed in AND has set his accessibility needs in his profile
- The accessibility calculations are based ONLY on the criteria selected in his profile

**Comment by joshi1983** [comment]
I assigned this to Blandine because she needs to finish writing this issue before anyone can work on it.

**Comment by joshi1983** [comment]
I made it so the overall rating calculation of a location in personalized mode is the average rating for the accessibility criteria that the user picks.  If the user hasn't picked any criteria, it is still the same as universal.

I defaulted the mode to 'Personal' on a location report when the user is signed in.
The location ratings in location search are also 'personal' whenever the users are signed in and have at least one criteria picked on their profiles.

**Comment by Blandine-AA** [comment]
@stephanosm, would you mind testing this. I'd like to have you input on the behavior. Thanks!

**Comment by Blandine-AA** [comment]
@stephanosm tested this and commented in #108 .

There is a problem here with the accessibility categories personal rating. The personal rating per categories is the same as the universal rating and it shouldn't. The overall rating for both personal and universal rating can be as simple as the average of the categories rating.
If a user is not interested in a category, then that category should not show on the personal view and rating.
The personal rating of each accessibility category should ONLY be based on the selected criteria in the user's profile.

**Comment by joshi1983** [comment]
Disable the personal view for now.

**Comment by Blandine-AA** [comment]
@joshi1983, is this done?

**Comment by joshi1983** [comment]
#652 is replacing the work from this issue with a different overhaul of the user interface.  It would be a waste to continue work on these older requirements when they'll be replaced soon.

For these reasons, I'm closing the issue.
